source/mainnet/net/web3-id/issuer.rst

```diff
@@ -0,0 +1,144 @@
+         "Some attribute": "Hello"
+       },
+       "id": "did:ccd:testnet:pkc:c162a48f58448234da9f3848dc3bc5fd7f2aa0e4b7e5e15654876365f8b86c1b"
+     },
+     "validFrom": "1970-01-01T00:00:00.017Z",
+     "validUntil": "1970-01-01T00:00:12.345Z",
+     "holderRevocable": true,
+     "metadataUrl": {
+       "url": "http://link/to/schema",
+       "hash": null
+     }
+   }
+
+``validUntil`` makes it possible for creditals to expire. ``validFrom`` allows you to set a start date and time for the credential, such as in the case of a backstage pass for a concert.
+
+An example response would be:
+
+.. code-block:: json
+
+   {
+     "txHash": "179de883eb0e748b05dcb3a3632302cea56d0f410df86a1cc4558f3274c1cf3e",
+     "credential": {
+       "credentialSchema": {
+         "id": "http://link/to/schema",
+         "type": "JsonSchema2023"
+       },
+       "credentialSubject": {
+         "attributes": {
+           "Another attribute": "World",
+           "Attribute 0": 1234,
+           "Some attribute": "Hello"
+         },
+         "id": "did:ccd:testnet:pkc:c162a48f58448234da9f3848dc3bc5fd7f2aa0e4b7e5e15654876365f8b86c1b"
+       },
+       "id": "did:ccd:testnet:sci:3:17/credentialEntry/c162a48f58448234da9f3848dc3bc5fd7f2aa0e4b7e5e15654876365f8b86c1b",
+       "issuer": "did:ccd:testnet:sci:3:17/issuer/",
+       "proof": {
+         "proofPurpose": "assertionMethod",
+         "proofValue": "facdb03a1d054a55808875864abc85cc41d2c32290929bbb361a710b0fda5e7f333ac33abdb1b5f0ebb5662335c34410b8e96ca6730df7eb100f814f223d0b07",
+         "type": "Ed25519Signature2020",
+         "verificationMethod": "did:ccd:testnet:pkc:7f9a19691d30963a13477da2e0e4ee5a78c61000eb36867141b519f003256f9b"
+       },
+       "randomness": {
+         "Another attribute": "6490531ea308a2e661f62c4678e00bb87c9f602be7a053e910f8e44609bc5adb",
+         "Attribute 0": "29b439aa58324b2be5c5a3ceb7ba23b48397ba1d1d9081869f56ff1c96a2b32f",
+         "Some attribute": "2f5e0279c8ff6bcb004024dd4ba4f3e29d30ec91e3e4583855c2dae35ae83f8d"
+       },
+       "type": [
+         "ConcordiumVerifiableCredential",
+         "UniversityDegreeCredential",
+         "VerifiableCredential"
+       ],
+       "validFrom": "1970-01-01T00:00:00.017Z",
+       "validUntil": "1970-01-01T00:00:12.345Z"
+     }
+   }
+
+If you have ``"holderRevocable": true,`` this means that the credential holder can revoke the credential themselves. If false, only the issuer can revoke the credential.
+
+Credential metadata controls how the credential is displayed in the wallet. An example is:
+
+.. code-block:: json
+
+    {
+      "title": "Mixed credential",
+      "logo": {
+        "url": "https://avatars.gibhubusercontent.com/u/39614219?s=200&v=4"
+      },
+      "backgroundColor": "#ffaa00"
+    }
+
+You can update the metadata for verifiable credentials, such as title, image, colors, or localization, but the credential information cannot be edited once the smart contract is deployed.
```

Comment:
"credential information cannot be edited" is a bit vague, since metadata is also "information".

Could we say "attributes" instead?